app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -90,203 +88,86 @@ For safeguarding reasons, schools will assess your suitability to teach from a r
  
-### If you do not have a university degree
-
-If you do not have a degree, you can [apply to study a 3 to 4 year undergraduate course](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) in England which combines a teaching degree with QTS. When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
-
-The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) has lots of helpful information about studying at an English university.
-
-### Further information
-
-[Find out more about training to be a teacher in England](/train-to-be-a-teacher). 
-
-Find out what financial support is available to [fund your training](/funding-and-support) if you are in the UK under the [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa), the [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) or the [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme).
-
-## Information for schools and training providers
-
-### Help from DfE
-
-From 1 February 2023, Ukrainians will be able to obtain QTS through DfE's new professional recognition service,[Apply for QTS in England](#apply-for-qts-in-england).
+You can also learn English with a private provider. Well respected English language courses include: 
 
-[Learn more about the new service](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers/a-fairer-approach-to-awarding-qts-to-overseas-teachers--2#). 
+* [International English Language Testing System (IELTS)](https://www.ielts.org/)
+* [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl)
+* [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)
 
-### Visas and immigration
+### Assessment only QTS
+If you find you do not meet the eligibility criteria to apply for QTS, you could consider assessment only QTS. This means you can work in an English school and, alongside your job, apply to an English teacher training provider who will assess if you meet the standards for English QTS.
 
-Most Ukrainian refugees will not need to be sponsored for a skilled worker or student visa. [Find out more about visa schemes for Ukrainian refugees, allowing them to live, work and study in the UK](#check-the-status-of-your-visa).
+Check with [assessment only training providers](/train-to-be-a-teacher/assessment-only-route-to-qts) for more information about eligibility.
 
-### Qualifications
+### Other teaching job options
```

Comment:
@laurahermionetennant92 not sure if this header quite works for all of the points below? 

For e.g. this one isn't a job option it's more about where they can look to find a job
[Find your local authority ('council')](https://www.gov.uk/find-local-council) and then search their website for teaching roles.

And Get school experience is an unpaid placement not a job
For a short-term unpaid placement, you can [request school experience in a primary or secondary school in England](https://schoolexperience.education.gov.uk/).